bindings/modbus/index.template.html

```diff
@@ -0,0 +1,338 @@
+                                    "readproperty",
+                                    "writeproperty"
+                                ],
+                                "modbus:entity": "Coil",
+                                "modbus:offset": 1,
+                                "modbus:length": 8
+                            }
+                </pre>
+        Another notable configuration of a form using the modbus vocabulary is the polling mechanism. Thanks
+        to the keyword <code>pollingTime</code> the user can indicate the intervals for observing a particular
+        set of registers. Supposing that the device knows that the value of coil register 1 does change every 
+        1000 ms, in [[[#example-polling]]], it suggest that the polling time should not be faster than 10 ms. WoT 
+        consumers may still create requests faster than the specified time but it should be taken as a reasonable
+        default for observing a property. 
+        <pre  id="example-polling" class="example" title="A complete TD example">
+            {
+                "href": "modbus+tcp://127.0.0.1:60000/1",
+                "op": [
+                    "observeproperty"
+                ],
+                "modbus:entity": "Coil",
+                "modbus:pollingTime: 1000,
+                "modbus:offset": 1
+            }
+        </pre>
+        Finally, [[[#full-td]]] shows a complete device described using modbus ontology.
+        <pre id="full-td" class="example" title="A complete TD example">
+                {
+                  "@context": [
+                      "https://www.w3.org/2019/wot/td/v1",
+                      {
+                          "modv": "https://www.example.com/ns/modbustcp"
+                      }
+                  ],
+                  "title": "ModbusPLC",
+                  "description": "An industrial machine, retrofitted to be IoT capable.",
+                  "id": "uri:dev:ModbusTCPThing",
+                  "securityDefinitions": {
+                      "nosec_sc": {
+                          "scheme": "nosec"
+                      }
+                  },
+                  "security": "nosec_sc",
+              
+                  "base": "",
+                  "properties": {
+                      "limitSwitch1": {
+                          "title": "downLimitSwitch",
+                          "type": "boolean",
+                          "description": "Limit switch moving downwards",
+                          "forms": [
+                              {
+                                  "op": "readproperty",
+                                  "href": "modbus+tcp://192.168.178.32:502/10003",
+                                  "modv:function": "2",
+                                  "modv:length": 1,
+                                  "modv:offset": "10003",
+                                  "modv:unitID": "1",
+                                  "contentType": "application/octet-stream"
```

Comment:
contentType should also include the order pattern of the bytes in the registers. E.g., lets assume the we receive the bytes in this order: 1 | 2 | 3 | 4 
Decoding a value (e.g., float) with little endian would lead to a reading sequence: 4 3 2 1
Sometimes the endians are only register wise: In that case we whould have an little endian order like 2 1 4 3 

I would suggest to introduce a parameter byteSeq which provides the encoded order of the registers. Eg.:

`"contentType": "application/octet-stream;byteSeq=1234"`